src/org/omegat/gui/preferences/view/HistoryAutoCompleterOptionsController.java

```diff
@@ -59,15 +59,15 @@ private void initGui() {
     @Override
     protected void initFromPrefs() {
         panel.historyCompletionCheckBox
-                .setSelected(Preferences.isPreference(Preferences.AC_HISTORY_COMPLETION_ENABLED));
+                .setSelected(!Preferences.isPreference(Preferences.AC_HISTORY_COMPLETION_ENABLED));
```

Comment:
```suggestion
                .setSelected(Preferences.isPreferenceDefualt(Preferences.AC_HISTORY_COMPLETION_ENABLED, true));
```

Reply:
`isPreferenceDefualt` should be `isPreferenceDefault`